Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:8.0
+
+ENV MYSQL_DATABASE=app_db \
+    MYSQL_USER=app_user \
+    MYSQL_PASSWORD=1234 \
+    MYSQL_ROOT_PASSWORD=rootpassword
+
+VOLUME /var/lib/mysql
```

Comment:
The `VOLUME` instruction syntax is incorrect. It should be `VOLUME ["/var/lib/mysql"]` to properly define the volume. This ensures that the data in `/var/lib/mysql` is persisted outside the container.